modules/investigating-kernel-crashes.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operating-system-issues.adoc
+
+[id="investigating-kernel-crashes"]
+= Investigating kernel crashes
+
+== Enabling `kdump`
+
+The `kdump` service, included in `kexec-tools`, provides a crash-dumping mechanism. You can use this service to save the contents of the system’s memory for later analysis.
+
+{op-system} ships with `kexec-tools`, but manual configuration is required to enable `kdump`.
+
+.Procedure
+
+The following steps are needed to enable `kdump` on {op-system}.
```

Comment:
Let's make this active voice. s/The following steps are needed to enable `kdump` on {op-system}./Perform the following steps to enable `kdump` on {op-system}:/